Dockerfile.local

```diff
@@ -28,6 +28,10 @@ RUN curl -o aws-iam-authenticator https://amazon-eks.s3-us-west-2.amazonaws.com/
 
 ENV PATH "$PATH:/usr/local/bin/aws-iam-authenticator"
 
+RUN wget -O /tmp/get-pip.py https://bootstrap.pypa.io/get-pip.py && \
+    python /tmp/get-pip.py && \
+    pip install --upgrade awscli==1.16.208
```

Comment:
The `aws eks get-token` command is available in version 1.16.232 or greater of the AWS CLI:
https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html

Reply:
I believe the AWS docs just links toward the latest versions (1.16.232 was released a few days ago, however the `aws eks get-token` has been around for a few months already 1.16.158): https://pypi.org/project/awscli/#history

I would keep this to 1.16.208 just to be in sync with clouddriver, which suffered from the same issue and was recently patched: https://github.com/spinnaker/clouddriver/pull/3997

What do you tink @clareliguori ?

Reply:
Hmm I'm just going off what the docs say, I don't work much on the EKS side of things

@micahhausler do you know what the story is here with the eks get-token command and AWS CLI versions?  Is it safe to lock to 1.16.208?